NEWS.txt

```diff
@@ -1,27 +1,66 @@
 ﻿           libSBML NEWS -- History of user-visible changes
 
 ====================================================================
-5.18.1 (Current svn)
+5.19.0
 ====================================================================
 
-STABLE RELEASE
+********************************************************************
+
+        The libSBML source code has moved to GitHub.
+            (https://github.com/sbmlteam/libsbml). 
+
+The stable branch reflect the latest released code. The development 
+branch is the main branch and we encourage users to create and 
+submit pull requests against this branch. We are no longer 
+maintaining a separate repository for experimental code for as yet 
+unfinished packages. 
+
+Please use the GitHub issue tracker to file any reports. We will gradually 
+move away from SourceForge, but for this release code will be 
+available from both SourceForge and GitHub. 
+
+
+
+* Addition of support for the SBML Distributions package
+
+  - Now that the first official version of the specification for the
+    SBML Level 3 Distributions package is available, the libSBML API for
+    'distrib' is now included in the stable release of libSBML.  This
+    means that all prebuilt binaries for the stable release will
+    include the 'distrib' package code. The src archive containing
+    libSBML core code will continue to be available, with additional
+    archives of the individual package code.
+    
+    NOTE: The libSBML GNU make-based build system has not been updated
+          for packages. Thus, to build from src, it is necessary to
+          use the CMake build system.
```

Comment:
maybe note, that this will be the last version that includes a gnumake build system, and that for the next version we'll rely on gnumake only?